msal/src/main/java/com/microsoft/identity/client/TokenParameters.java

```diff
@@ -137,6 +140,10 @@ public AccountRecord getAccountRecord() {
         return mAccountRecord;
     }
 
+    public String getCorrelationId() {
```

Comment:
Javadoc public methods.

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1063/commits/6a71d1d9fc3d3329f4b608dd3702a39ff7099b62